src/modules/core/components/DropdownMenu/DropdownMenu.css

```diff
@@ -6,7 +6,7 @@
   display: block;
 }
 
-.baseStyles button {
+.baseStyles ul > li > button {
   /* Reset button styles to look like a link in here */
```

Comment:
This is so the "manage tokens" button, which is not nested within the ul > li > button dom structure, does not have its styles reset.